README.md

```diff
@@ -1 +1,46 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HELLO-MICROVERSE
+
+> Hello-microverse is a simple project build with HTML/CSS only.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- Please, let's make our readme more professional by updating this section and adding some instructions on how users can have your project up and running locally on their machines. e.g: How to clone the repo, etc.

Reply:
correction done